docs/getting-started/overview.rst

```diff
@@ -1,6 +1,92 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implemetentation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Base Class Library** or BCL for short) that supports 
```

Comment:
The BCL is the bottom part of the libraries, not the whole thing. Jeff Richter talks about BCL and FCL in his books.